.devcontainer/postCreate.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# Install dev dependencies from OPAM
+opam init -y --bare --disable-sandboxing git+https://github.com/rescript-lang/opam-repository
+opam switch create 5.2.0 --packages ocaml-option-static
+opam install . --deps-only -y
+
+# For IDE support, install the OCaml language server
+opam install ocaml-lsp-server -y
+
+# Add OPAM environment setup to shell startup script
+echo 'eval $(opam env)' >> ~/.zshrc
```

Comment:
So this is working for you when using zsh, i.e., you don't have to do run `eval $(opam env)` manually, right?

Maybe we should do this for bash, too?

Reply:
Yes, like I said. Bash isn't working for me either but this line makes it work for every new `zsh` I open. Will add line for bash.

Reply:
![image](https://github.com/user-attachments/assets/7d2587d1-2d24-45e2-b90c-0d70ada4ebf4)

this worked for me.